### PR TITLE
[Trivial] Delete duplicated file @open sesame 11/19 10:09

### PR DESCRIPTION
**Pull request by zhoonit**
## Commits to be reviewed in this PR

<details><summary>[Trivial] Delete duplicated file</summary><br />

This patch delete duplicated test file that is not used anywhere

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

</details>

### CODE REVIEW

File: test/unittest/layers/unitest_layer_node.cpp [hunk 1]
```diff
@@ -1,94 +0,0 @@
-// SPDX-License-Identifier: Apache-2.0
-/**
- * Copyright (C) 2020 Parichay Kapoor <pk.kapoor@samsung.com>
- *
- * @file        unittest_nntrainer_layers.cpp
```
Comment (zhoonit) [inline test/unittest/layers/unitest_layer_node.cpp:5]: There is unit**t**est_layer_node.cpp although it's also not used anywhere.

**Review by myungjoo — APPROVED** [review]

**Comment by jaeyun-jung** [comment]
Need discussion later. 
To run the unittests on gbs-build, we should block tizen-feature for ML API. Internally ml-api (api repository, nnstreamer only now) disables feature state while running unittests.
If nntrainer uses ml-common APIs (tensors-info and tensors-data), unittests in nntrainer always report failure because common ML API cannot check the feature state on gbs-build.

**Comment by zhoonit** [comment]
> Need discussion later.
> To run the unittests on gbs-build, we should block tizen-feature for ML API. Internally ml-api (api repository, nnstreamer only now) disables feature state while running unittests.
> If nntrainer uses ml-common APIs (tensors-info and tensors-data), unittests in nntrainer always report failure because common ML API cannot check the feature state on gbs-build.

I have moved the comment to https://github.com/nnstreamer/api/issues/110 for better discussion, the issue addressed is not related to the PR :)

**Review by lhs8928 — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**